src/game/stage/stage.c

```diff
@@ -47,9 +47,9 @@ void SetupStageIntro(void);
 
 void CreateStageWaterTask(u32, u32, u32);
 
-void sub_8019120(void);
+void CreateMultiplayerSendEventMgr(void);
 
-void sub_80191A4(void);
+void CreateMultiplayerRecieveEventMgr(void);
```

Comment:
Don't have to do this, might as well move these to their headers? If they are already there then including the headers and removing these would be good.